main.tf

```diff
@@ -103,13 +104,16 @@ module "google_mysql_db" {
         ipv4_enabled        = var.public_access_read_replica
         private_network     = var.private_network
         require_ssl         = null
+        allocated_ip_range  = var.read_replica_pvt_ip_range
       }
-      database_flags      = local.db_flags_read_replica
-      disk_autoresize     = var.disk_auto_resize_read_replica
-      disk_size           = var.disk_size_gb_read_replica
-      disk_type           = "PD_SSD"
-      user_labels         = var.labels_read_replica
-      encryption_key_name = null
+      database_flags        = local.db_flags_read_replica
+      disk_autoresize       = var.disk_auto_resize_read_replica
+      disk_autoresize_limit = var.disk_autoresize_limit
+      disk_size             = var.disk_size_gb_read_replica
+      disk_type             = "PD_SSD"
+      availability_type     = var.read_replica_availability_type
```

Comment:
Make the variable name as `availability_type_read_replica` to make it consistent with variable naming convention.

Reply:
Or better yet, follow the master node. Something like:
`var.highly_available_read_replica ? "REGIONAL" : "ZONAL"`

Reply:
Added as requested